docs/Trainer/Checkpointing.md

```diff
@@ -1,6 +1,7 @@
 Lightning can automate saving and loading checkpoints.
 
 ---
+
 ### Model saving
 Checkpointing is enabled by default to the current working directory.
 To change the checkpoint path pass in :
```

Comment:
Can you explain all the other options? A table would be good.

-1, 1, k

Reply:
I added a table. Meanwhile, I strongly suggest to use some docstrings instead of writing the same things twice in comments and "docs". When editing it is disaster